src/deployment/deployment.t.sol

```diff
@@ -96,7 +96,8 @@ contract VatDeployer {
 
         spotter.poke("ETH-A");
 
-        pool = new Pool(address(vat), address(0x12345678), address(spotter), address(new FakeJug()));
+        address fakeDaiToUsdPriceFeed = address(0);
```

Comment:
this should be fixed, but let's leave it for now, will open an issue

Reply:
anw fixed it